src/Pages/MainPage/index.jsx

```diff
@@ -63,34 +74,20 @@ class MainPage extends Component {
           />
         ) : (
           <div>
-            <div className="mainPage__newPost" onClick={this.toggleNewPost}>
-              What are you thinking now?
-            </div>
             <div className="mainPage__postList">
-              {!feedLoading && this.props.user && feed
-                ? feed.map((data) => {
-                    return (
-                      <div className="mainPage__post mb-3" key={data.id}>
-                        <Post
-                          postId={data.id}
-                          isMyPost={this.props.user.id === data.UserId}
-                          userProfileImg={data.User.profileImgName}
-                          userName={data.User.name}
-                          userType={data.User.type}
-                          postContent={data.content}
-                          postHashTags={[
-                            { id: 1, tags: 'flower' },
-                            { id: 2, tags: 'sunny' },
-                          ]} //////
-                          postImg={data.postImgName}
-                          postDate={data.createdAt}
-                          isLiked={data.isLiked}
-                          postLike={data.likeCount}
-                        />
-                      </div>
-                    );
-                  })
-                : null}
+              {dummyPost.posts.map((data) => {
+                return (
+                  <Post
+                    postId={data.id}
+                    postImg={data.postImgName}
+                    postTitle={data.postTitle}
+                    isFavorite={data.isFavorite}
+                    userId={data.UserId}
+                    key={data.id}
```

Comment:
이 부분 예전에 생각해봤었는데 data를 한 번에 주고 넘기는 건 어떤가유? 
아니면 이렇게 하는 방식이 더 편한감?
```suggestion
                    key={data.id}
                    postData={data}
```